README.rst

```diff
@@ -166,9 +166,17 @@ NOTE: the output for the following formats are generated in accordance with XML
 * ``junit`` follows the `JUnit xml schema from Jenkins </doc/junit-10.xsd>`_
 * ``checkstyle`` follows the common `"checkstyle" xml schema </doc/checkstyle.xsd>`_
 
+The ``--quiet`` Do not output any message. 
 
 The ``--verbose`` option will show the applied rules. When using the ``txt`` format it will also display progress notifications.
 
+NOTE: if there is an error like "errors reported during linting after fixing", you can use this to be even more verbose for debugging purpose
+
+* ``--verbose=0`` or no option: normal
+* ``--verbose``, --verbose=1, -v: verbose
```

Comment:
Don't we want `--verbose=1` and `-v` in backticks too?

Reply:
yep, updated.